modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +63,65 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  checkNAMES();
+  /**
+   *this function will consantly check for {USERNAMES} and {FULLNAMES}
+   */
+  function checkNAMES(){
+    var x=0,y=-1,z=-1;    //y for {FULLNAMES}  z for {USERNAMES}
+    for(x=0;x<length;x++){

```

Comment:
No, I don't think an alert is necessary. There is help text there that says, "Setting a single option of '{FULLNAMES}' or '{USERNAMES}' will produce a list of users based on the group limits set below." and this should make it clear that those special options only work when they are the only items set in the list. As long as the option above which prevents adding new options is enforced, then it will be fine.